CriteoPublisherSdk/Sources/CR_BidManager.m

```diff
@@ -179,6 +180,47 @@ - (void)prefetchBidsForAdUnits:(CR_CacheAdUnitArray *)adUnits {
          }];
 }
 
+- (void)fetchLiveBidForAdUnit:(CR_CacheAdUnit *)adUnit
+           bidResponseHandler:(CR_BidResponseHandler)responseHandler {
+  [self fetchLiveBidForAdUnit:adUnit
+           bidResponseHandler:responseHandler
+                   timeBudget:CRITEO_DEFAULT_LIVE_BID_TIME_BUDGET_IN_SECONDS];
+}
+
+- (void)fetchLiveBidForAdUnit:(CR_CacheAdUnit *)adUnit
+           bidResponseHandler:(CR_BidResponseHandler)responseHandler
+                   timeBudget:(NSTimeInterval)timeBudget {
+  [self.threadManager dispatchAsyncOnGlobalQueueWithTimeout:timeBudget
```

Comment:
Don't you need to prevent the request if silenced ? Either with the user silence or with slot silence.

Reply:
Yes this is needed, this will be taken by the upcoming refactorings.

Reply:
Fixed partially by dc08fc6
Also silent slot isn't handled yet, FAR if that is fine with you.